network-api/networkapi/wagtailpages/templates/bg_about.html

```diff
@@ -1,4 +1,4 @@
-{% extends "./bg_base.html" %}
+{% extends "buyersguide/bg_base.html" %}
```

Comment:
`./bg_base.html` was the old buyersguide base template. It should have been using the new namespaced bg_base.html template. So it is now.